src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_NUMBER = 100;
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        String randomColor = String.valueOf(colorSupplier.getRandomColor());
+        int randomNumber = new Random().nextInt(MAX_NUMBER) + 1;

```

Comment:
![image](https://github.com/mate-academy/jv-lottery/assets/20326858/d4ea6a41-e2c9-4395-8edc-72c2c52fd696)

Reply:
Hi ! Sorry, I don't understand what do you mean? Please can you explain a little more?